bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
+    if (money >= 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.3)
+        console.log('Lương ròng: '+money*0.7)
+    }
```

Comment:
```suggestion
  if (money >= 15000000) {
      console.log('Thuế thu nhập: ' + money * 0.3)
      console.log('Lương ròng: ' + money * 0.7)
  }
```
Hơn nữa công thức đúng phải là lương thực nhận = tổng lương - thuế